packages/types/src/userOrganizationInvitation.ts

```diff
@@ -25,7 +25,7 @@ export interface UserOrganizationInvitationResource extends ClerkResource {
     imageUrl: string;
     name: string;
     id: string;
-    slug: string;
+    slug: string | null;
```

Comment:
I didn't properly type this one. Organizations have a slug of string | null so the slug of `UserOrganizationInvitationResource` needs to match that

Reply:
@panteliselef, @desiprisg since the response of the BAPI/FAPI is defined as `slug: string` i think we should update the type to have it the same.
If there are cases where the `Organization` type is used as part of a request param, I would suggest we update it to another type to avoid confusion.
Do you think the refactoring is feasible or am i missing something? If you agree with my suggestion could you open a ticket to refactor this?